ansible/roles/sufia/tasks/data-repo.yml

```diff
@@ -64,6 +64,6 @@
     ignore_errors: True
 
   become: yes
-  become_user: '{{ project_user }}'
+  become_user: '{{ project_runner }}'
```

Comment:
It probably made sense to me to use `project_runner` there as that is the user under which the Rails app will ultimately run.  However, I now believe this is likely incorrect and a victim of a lack of testing on my part.  Lots of the actions done in the `block` have `ignore_errors: True` and are optional.  I don't have a `user_list.txt` nor an `admin_list.txt` and so didn't test those code paths.  Also, I don't have any carousel images `images.zip` file and so wasn't able to test that code path either.

Looking at the `block` of actions, it does seem more reasonable to execute these as `become_user: '{{ project_runner }}'`.  Did that work for you the other day?